Xamarin.PropertyEditing.Mac/Controls/StringEditorControl.cs

```diff
@@ -1,18 +1,31 @@
 using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Diagnostics;
+using System.Linq;
 using AppKit;
 using CoreGraphics;
+using Foundation;
 using Xamarin.PropertyEditing.Mac.Resources;
 using Xamarin.PropertyEditing.ViewModels;
 
 namespace Xamarin.PropertyEditing.Mac
 {
 	internal class StringEditorControl : PropertyEditorControl<PropertyViewModel<string>>
 	{
+		private NSTextField stringEditor { get; set; }
+
+		private NSLayoutConstraint stringEditorWidthConstraint;
+
+		public override NSView FirstKeyView => this.stringEditor;
+		public override NSView LastKeyView => this.stringEditor;
+
+		internal NSPopUpButton inputModePopup;
+		private IReadOnlyList<InputMode> viewModelInputModes;
```

Comment:
If we're going to end up moving stuff like this, let's move it to the consistent place

Reply:
In most of the controls/editors on Mac, the declarations are at the top. Hence why I moved these there.

Reply:
Right but that's not where they are on Windows or Core.